org-ql.el

```diff
@@ -825,6 +825,47 @@ A and B are Org headline elements."
             (a-priority t)
             (b-priority nil)))))
 
+;;;; Functions/Occur
+
+(defun org-ql-occur (query &optional keep-previous callback)
+  "Make a compact tree which shows all matches of the query.
+
+QUERY is an `org-ql' query sexp (quoted, since this is a
+function).
+
+The tree will show the lines where the query matches, and any other context
+defined in `org-show-context-detail', which see.
+
+When optional argument KEEP-PREVIOUS is non-nil, highlighting and
+exposing done by a previous call to `org-occur' or `org-ql-occur'
+will be kept, to allow stacking of calls to this command.
+
+Optional argument CALLBACK can be a function of no argument.  In this case,
```

Comment:
I think we can omit the `callback` argument, because the `org-ql` query performs the same task.

Reply:
You are right.